Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/ColliderRotationManipulators.cpp

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzFramework/Viewport/ViewportColors.h>
+#include <AzToolsFramework/ViewportSelection/EditorSelectionUtil.h>
+#include <EMotionFX/CommandSystem/Source/ColliderCommands.h>
+#include <EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/RenderPlugin/ViewportPluginBus.h>
+#include <EMotionStudio/EMStudioSDK/Source/EMStudioManager.h>
+#include <Editor/ColliderContainerWidget.h>
+#include <Editor/Plugins/Ragdoll/ColliderRotationManipulators.h>
+
+namespace EMotionFX
+{
+    ColliderRotationManipulators::ColliderRotationManipulators()
+        : m_rotationManipulators(AZ::Transform::Identity())
+    {
+        m_rotationManipulators.SetCircleBoundWidth(AzToolsFramework::ManipulatorCicleBoundWidth());
+        m_adjustColliderCallback = AZStd::make_unique<PhysicsSetupManipulatorCommandCallback>(false);
+        EMStudio::GetCommandManager()->RegisterCommandCallback("AdjustCollider", m_adjustColliderCallback.get());
```

Comment:
As there is just one callback here it doesn't matter but just in case you haven't seen this yet, there is a templated version where you can register the callback + add it to a callbacks array etc. in a single line:

`GetCommandManager()->RegisterCommandCallback<MyCallback>("MyCommand", m_callbacks, args);`

Reply:
OK, good to know, thanks